runtime/cudaq/platform/pasqal/PasqalServerHelper.h

```diff
@@ -6,14 +6,17 @@
  * the terms of the Apache License 2.0 which accompanies this distribution.    *
  ******************************************************************************/
 
+#include "common/RestClient.h"
 #include "common/ServerHelper.h"
 
 namespace cudaq {
 
-class PasqalServerHelper : public ServerHelper {
+class PasqalServerHelper : public ServerHelper {  
 protected:
+  /// @brief Pasqal's REST API.
   const std::string baseUrl = "https://apis.pasqal.cloud";
-  
+  const std::string apiPath = "/core-fast/api/v1";
```

Comment:
actually the `v1` should be a part of the path of the operation, as we already have a few endpoints on v2.